int64 id
bool routine
int64 id_routine
string name_mg
int64[] trans_alphabet
float64 dtimes
